Kernel/VM/MemoryManager.cpp

```diff
@@ -78,8 +85,10 @@ void MemoryManager::initialize_paging()
     // 8 MB   -> MAX            Userspace physical pages (available for allocation!)
 
     // Basic virtual memory map:
-    // 0 MB   -> 8MB            Identity mapped.
-    // 0xc0000000-0xffffffff    Kernel-only virtual address space.
+    // 0x00000000-0x00100000    Identity mapped for Kernel Physical pages
+    // 0x00800000-0xbfffffff    Userspace program virtual address space
+    // 0xc0001000-0xc0800000    Kernel mapping, encompassing kmalloc, etc 
+    // 0xc0800000-0xffffffff    Kernel-only virtual address space. This area is mapped to the first 8 MB of physical memory
```

Comment:
I don't understand the comment here. How can this huge range be mapped to only 8MB of physical memory?

Reply:
Typo sorry. This was meant for the line above.